lib/vfscore/pipe.c

```diff
@@ -81,6 +82,11 @@ struct pipe_file {
 	int r_refcount;
 	/* Flags */
 	int flags;
+
+	/* Eventpoll lock */
+	struct uk_mutex eplock;
+	/* List of registered eventpolls */
+	struct uk_list_head eplist;
```

Comment:
As I mentioned in the other PR, I'm in favor of using `evp` prefix instead of `ep`, it seems more intuitive to me. But it's a personal preference.